frontend/src/components/ConnectedAccountsPage.jsx

```
@@ -11,14 +11,51 @@ import {
+        setSearchParams(searchParams, { replace: true });
+      }, 10000);
+      return () => clearTimeout(timer);
+    }
+  }, [highlightPlugins, searchParams, setSearchParams]);
+
+  // Scroll to the highlighted element once it's mounted (depends on plugin data loading)
+  useEffect(() => {
+    if (highlightPlugins.size > 0 && highlightRef.current) {
+      highlightRef.current.scrollIntoView({ behavior: 'smooth', block: 'center' });
+    }
+  }, [highlightPlugins, pluginsQ.data]);
```

Biome (2.3.14)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)